src/sniffer/pkg/ipresolver/procfs_resolver.go

```diff
@@ -0,0 +1,206 @@
+package ipresolver
+
+import (
+	"errors"
+	"github.com/otterize/network-mapper/src/sniffer/pkg/utils"
+	"github.com/sirupsen/logrus"
+	"sync"
+)
+
+type ProcessMonitor struct {
+	processes  map[int64]interface{}
```

Comment:
Should replace with set

Reply:
AFAIU, this is the standard way to have a "set" in Golang

Reply:
We have a set library, we use it in the mapper code. Also K8s operator SDK has one. Not a blocker, will deal with those stuff later